plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -88,6 +88,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
 
     @Test
     @Override
+    @org.junit.jupiter.api.Test
```

Comment:
Doesn't it mean these tests run twice, once by TestNG and once by JUnit?

Reply:
I have no idea how it works right now tbh :)

Reply:
Remove the testng annotation. This is a test that has already been migrated to junit

Reply:
How can I tell which tests were migrated?

Reply:
Look at the annotation on the test

Reply:
A test should generally have exactly one `@Test` annotation (unless it's `@Override` of a testng test) and having one `@Test` annotation  should be sufficient for it to be run.

So if bigquery overrides a test method, it shouldn't matter which ``@Test`` annotation it uses, the test will still run.
Since we're migrating testng->junit migration, there is a rather low risk a test will end up running twice. The only real risk is that test is not run at all, but again, having one whichever `@Test` annotation should be sufficient.

That's a very practical bar. I don't want to be in a business of going up and down class hierarchies and checking _which_ `@Test` annotation it is. If we had to do this, we would need to bad imports of these annotations and always use FQNs.